README.md

```diff
@@ -14,6 +14,20 @@ Although this project is very new, it has been thoroughly tested in all modern b
 
 __[Demo](http://jakesidsmith.github.io/react-reorderable/)__
 
+## Installation
+
+* Using npm
+  ```
+  npm install react-reorder
+  ```
+    Add `--save` or `-S` to update your package.json
+
+* Using bower
+  ```
+  bower install react-reorderable
```

Comment:
Do you think it's worth aligning on bower? Seems like it may cause confusion down the line to have two names.